### DISCUSSION TITLE
Curation report Feb 27 - March 10

### DISCUSSION THREAD

**Discussion by IsabelWassink** [wikipathways/GPML-Repo#118] [Curation reports]
I did the curation for two weeks since Denise did not have time to look at all the steps last week. 

- **Recently changed pathways:** Lots of recent changes (mostly new or already approved pathways) and 12 new pathways
   - some new and some test/academy pathways (shouldn't [WP5332](https://classic.wikipathways.org/index.php/Pathway:WP5332) be tagged as a tutorial pathway (@khanspers)? 
   - several pathways related to DMD (some seem to be overlapping) - are they part of a student project? [WP5341](https://classic.wikipathways.org/index.php/Pathway:WP5341), [WP5335](https://classic.wikipathways.org/index.php/Pathway:WP5335), [WP5337](https://classic.wikipathways.org/index.php/Pathway:WP5337), [WP5339](https://classic.wikipathways.org/index.php/Pathway:WP5339), [WP5340](https://classic.wikipathways.org/index.php/Pathway:WP5340)
   - some of the users seem to use Ensembl Human instead of Ensembl or UniProt-SwissProt instead of UniProt-TrEMBL annotations and I also see some weird encodings with special characters and references
- **Under construction**: no changes required
- **Tutorial**: one pathway already tagged with required deletion (I don't have the permissions to do that) [WP5338](https://classic.wikipathways.org/index.php/Pathway:WP5338)
- **Stub pathways**: should WP5308 be deleted since it hasn't been updated since the tag was left?
- **Update approval/featured tags**: all are up to date
- The search works (on the classic site)

@jmillanacosta is up next

**Comment by mkutmon** [comment]
@tabbassidaloii, are the DMD related pathways from your students?

**Reply by tabbassidaloii** [reply]
@mkutmon, Yes. I don't intend to approve those pathways. My next student will integrate them into one pathway to avoid repetitions. I hope that is not a problem.

**Reply by DeniseSl22** [reply]
I've added the 'test pathway' tag for clarity, and a comment stating these will be merged in the future

**Comment by egonw** [comment]
> * some of the users seem to use Ensembl Human instead of Ensembl or UniProt-SwissProt instead of UniProt-TrEMBL annotations

We routinely test these and I'll fix any remaining errors next week.

**Reply by DeniseSl22** [reply]
I've resolved the SwissProt entries, couldn't find the 'Ensembl Human' ones...

**Reply by egonw** [reply]
I fixed the last few 'Ensembl Human' earlier this morning :)

**Reply by IsabelWassink** [reply]
Thank you!

**Comment by DeniseSl22** [comment]
> * some of the users seem to use Ensembl Human instead of Ensembl or UniProt-SwissProt instead of UniProt-TrEMBL annotations and I also see some weird encodings with special characters and references

@IsabelWassink : that's something which should be fixed in PV4 (ask @mkutmon for details).

**Comment by khanspers** [comment]
@IsabelWassink : Thank you for a great report, and nice work on your first curation round! For WP5332, it was a duplicate that I had used for testing and had forgotten to delete. Just removed it now.
